torchci/test/mergeBot.test.ts

```diff
@@ -50,11 +50,11 @@ describe("merge-bot", () => {
 
   test("quoted merge/revert command no event", async () => {
     const merge_event = require("./fixtures/issue_comment.json");
-    merge_event.payload.comment.body = "> @pytorchbot merge this";
+    merge_event.payload.comment.body = "> @pytorchbot merge";
     const revert_event = require("./fixtures/issue_comment.json");
-    revert_event.payload.comment.body = "> @pytorchbot revert this";
```

Comment:
Can we add a negative test that "merge this"/"revert this" would result in some sort of error